state/export_test.go

```diff
@@ -6,6 +6,9 @@ import (
 	vmcommon "github.com/ElrondNetwork/elrond-vm-common"
 )
 
+// LastSnapshotStarted -
+const LastSnapshotStarted = lastSnapshotStarted
```

Comment:
[question]

Why create an extra exported variable(`LastSnapshotStarted`)  instead of exporting the existing one(`lastSnapshotStarted`) ?

Reply:
I wanted exported only in tests. Will be exported on the package only if required elsewhere

Reply:
Right, my bad, I thought `lastSnapshotStarted` is from a test package as well